code/modules/meteors/meteors.dm

```diff
@@ -2,7 +2,7 @@
 #define MAP_EDGE_PAD 5
 
 GLOBAL_VAR_INIT(meteor_wave_delay, 625) //minimum wait between waves in tenths of seconds
-//set to at least 100 unless you want evarr ruining every round
+//set to at least 100 unless you want evarr ruining every round (This spelling mistake is older then git, I'm scared to touch it)
```

Comment:
>changes the git blame by adding something to the same line

Reply:
Thank you for the catch, I'll move it to the line below.
I don't think the initial blame itself is that old, but may as well preserve